janusgraph-core/src/main/java/org/janusgraph/graphdb/management/JanusGraphManager.java

```diff
@@ -252,6 +254,8 @@ public Graph openGraph(String gName, Function<String, Graph> thunk) {
     @Override
     public Graph removeGraph(String gName) {
         if (gName == null) return null;
+        // Remove the graph from the script engine bindings so that closed/dropped graph instances do not leak
+        removeBinding(gName);
```

Comment:
Shouldn't we convert `gName` to `tsName`? I.e. `removeBinding(gName + "_traversal");`

Reply:
In this case, we need to refer the graph itself so I believe it should be `gName`.

Reply:
You right, thanks!